src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -35,6 +35,10 @@
 TEST_DIR = os.path.join(
     os.path.dirname(os.path.realpath(__file__)), 'corpus_pruning_task_data')
 
+TEST_BUCKET_GLOBAL = 'clusterfuzz-test-global-bundle'
```

Comment:
could do GLOBAL, SHARED, etc in middle to match like TEST_BLOBS_BUCKET, TEST_CORPUS_BUCKET, etc.

Reply:
Done!